packages/one-dark-ui/lib/main.js

```diff
@@ -9,13 +9,6 @@ module.exports = {
     atom.config.observe(`${themeName}.hideDockButtons`, setHideDockButtons);
     atom.config.observe(`${themeName}.stickyHeaders`, setStickyHeaders);
 
```

Comment:
You accidentally deleted a brace
```suggestion
  },

```